manim/mobject/geometry/line.py

```diff
@@ -930,7 +932,7 @@ def __init__(
         line1: Line,
         line2: Line,
         radius: float | None = None,
-        quadrant: Point2D = (1, 1),
+        quadrant: tuple[Literal[-1, 1], Literal[-1, 1]] = (1, 1),
```

Comment:
lol this gave me a chuckle when I saw it :)

What are your thoughts about moving this into a `TypeAlias` to explain what the values mean?

Reply:
I agree with you, and defined an `AngleQuadrant` type alias with some documentation.